src/app/servers/header/header.component.ts

```diff
@@ -6,10 +6,27 @@ import { Component, OnInit } from '@angular/core';
   styleUrls: ['./header.component.css']
 })
 export class HeaderComponent implements OnInit {
-
+  usernaeme: any;
+  storageUsers: any;
+  currentUserName: any;
   constructor() { }
 
   ngOnInit() {
+    this.currentUserName = JSON.parse(localStorage.getItem('currentUser')).name;
```

Comment:
You should have used localStorage service (service to work with localstorage). You should never add `JSON.parse(localStorage.getItem ` stuff in any component but instead use storageService.get('currentUser')